src/mark.ts

```diff
@@ -27,3 +30,306 @@ export const SQUARE = Mark.SQUARE;
 
 export const ERRORBAR = Mark.ERRORBAR;
 export const PRIMITIVE_MARKS = [AREA, BAR, LINE, POINT, TEXT, TICK, RULE, CIRCLE, SQUARE];
+
+export type FontStyle = 'normal' | 'italic';
+export type FontWeight = 'normal' | 'bold';
+/**
+ * @type integer
+ * @minimum 100
+ * @maximum 900
+ */
+export type FontWeightNumber = number;
```

Comment:
I see. Maybe related to https://github.com/YousefED/typescript-json-schema/issues/57